bicep/ccw.bicep

```diff
@@ -264,12 +265,12 @@ module oodNIC 'ood-NIC.bicep' = if (deployOOD) {
 }
 
 // create a user assigned managed identity to be assigned to the OOD VM
-resource oodNewManagedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' = if (deployOOD) {
+resource oodNewManagedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' = if (deployOOD && registerOODApp) {
```

Comment:
As registerOODApp is only true when deploying OOD this test can be simplified by testing only regiterOODApp